package.json

```diff
@@ -18,12 +18,17 @@
     "build/*"
   ],
   "devDependencies": {
+    "babel-eslint": "^8.2.6",
     "classnames": "^2.2.6",
     "eslint": "^4.9.0",
-    "eslint-config-airbnb": "^17.0.0",
-    "eslint-config-synacor": "^2.0.2",
+    "eslint-config-standard": "^11.0.0",
```

Comment:
In future try to separate the maintenance part as separate pull request. Then anyone else / any other pull request depending on it does not have to wait for the feature to be merged.

Reply:
Sure!